### PR TITLE
Integration-tests: cover the global metrics in corecheck docker

### PR DESCRIPTION
**Pull request by JulienBalestra**
### What does this PR do?

Integration-tests: Cover the global metrics in corecheck docker by checking if the `docker.images` and `docker.containers` are correct.

### CODE REVIEW

File: test/integration/corechecks/docker/globalmetrics_test.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator/mocksender"
+	"github.com/stretchr/testify/mock"
+)
+
+func init() {
+	registerComposeFile("globalmetrics.compose")
+}
+
+func isGreaterOrEqual(atLeast float64) interface{} {
```
Comment (xvello) [inline test/integration/corechecks/docker/globalmetrics_test.go]: Please move it to mocksender so it can be reused

**Review by xvello — COMMENTED** [review]
Logic LGTM, let's just move isGreaterOrEqual to the mocksender package

File: pkg/aggregator/mocksender/asserts_test.go [hunk 2]
```diff
@@ -18,35 +18,49 @@ type unittestMock struct {
...
+
+	const dummyValue = 3.0
+	m.dummyMethod(dummyValue)
+
+	for i := 0.0; i < dummyValue+1; i++ {
+		m.AssertCalled(t, "dummyMethod", IsGreaterOrEqual(i))
+	}
+	m.AssertNotCalled(t, "dummyMethod", IsGreaterOrEqual(dummyValue+1))
+
+	// Create a local testing.T just for the following AssertCalled
```
Comment (JulienBalestra) [inline pkg/aggregator/mocksender/asserts_test.go]: Not sure if it's relevant to check the fail status on `localTester testing.T`

Comment (xvello) [reply pkg/aggregator/mocksender/asserts_test.go]: Let's keep it, to test for false positives

**Review by xvello — APPROVED** [review]
LGTM

**Pull request merged**